internal/policies/client.go

```diff
@@ -403,7 +404,8 @@ func (f *Client) getPolicyServerURLRunningPolicy(ctx context.Context, policy pol
 	if f.policyServerURL != "" {
 		url, err := url.Parse(f.policyServerURL)
 		if err != nil {
-			log.Fatal().Msg("incorrect URL for policy-server")
+			f.logger.Error("incorrect URL for policy-server")
+			os.Exit(1) // TODO: is it OK to exit here when using Error instead of Fatal?
```

Comment:
@fabriziosestito PTAL if this makes sense. Do we want to:
1. Keep this as-is (I'll remove the `TODO`)
2. Create a Fatal log level for the slog based logger?
3. Use `log` from std library?
4. Something else that I'm not able to think of?